### PR TITLE
mosquitto: update to 1.6.3.

### PR DESCRIPTION
**Pull request by damadmai**
Also added runit service as unprivileged user for it

### CODE REVIEW

**Comment by jnbr** [comment]
Please change the system account to `_mosquitto` to prevent conflicts with names of user accounts.

File: srcpkgs/mosquitto/template [hunk 1]
```diff
@@ -9,12 +9,16 @@ make_check_target=test
...
+short_desc="MQTT version 3.1/3.1.1/5.0 compatible message broker"
 maintainer="Lukas Braun <koomi@hackerspace-bamberg.de>"
 license="EPL-1.0, BSD-3-Clause-Attribution"
-homepage="http://mosquitto.org"
+homepage="https://mosquitto.org"
 distfiles="${homepage}/files/source/${pkgname}-${version}.tar.gz"
-checksum=bd730d461f5f0adf6740abf2424c76c6d1263db0011fbb073c7a5c7eb8cc188b
+checksum=9ef5cc75f4fe31d7bf50654ddf4728ad9e1ae2e5609a4b42ecbbcb4a209ed17e
+
+system_accounts="_mosquitto"
```
Comment (xtraeme) [inline srcpkgs/mosquitto/template]: You've changed it to `_mosquitto` but later you use `mosquitto_var`.

File: srcpkgs/mosquitto/files/mosquitto/run [hunk 2]
```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+
+cd /var/lib/mosquitto
+exec chpst -u mosquitto:mosquitto mosquitto -c /etc/mosquitto/mosquitto.conf
```
Comment (xtraeme) [inline srcpkgs/mosquitto/files/mosquitto/run]: `_mosquitto`

File: srcpkgs/mosquitto/template [hunk 3]
```diff
@@ -1,6 +1,6 @@
 # Template file for 'mosquitto'
 pkgname=mosquitto
-version=1.6.0
+version=1.6.3
 revision=2
```
Comment (xtraeme) [inline srcpkgs/mosquitto/template]: revision must be bumped

Comment (Hoshpak) [reply srcpkgs/mosquitto/template]: why? It's a new version so usually the revision would be reset to 1.

**Comment by damadmai** [comment]
I got an E-Mail with a comment from @Hoshpak  which I don't see here in the conversation view:
> revision=2
> why? It's a new version so usually the revision would be reset to 1.
Everytime I made a mistake in a pull request I was also asked to increase the revision. I think this is for testing purposes for the reviewer. The new revision makes sure that the package gets updated.

**Comment by Hoshpak** [comment]
The conversation was already marked as resolved. I reverted that so it will be visible again. I asked because I'm not sure why @xtraeme requested a revision bump this time. Usually when you change a template without increasing the version, the revision needs to be bumped for the builders to pick it up and rebuild the package.

However when you update a package, the higher version number is enough for the builders to pick it up and the revision is reset to 1 since this is the first revision of the template at this version.

The only exception is if a package has been updated before and the update was reverted, the revision needs to be increased in this case to be higher than the previous on when you update it again.

File: srcpkgs/mosquitto/template (hunk 3 above)
Comment (xtraeme) [reply srcpkgs/mosquitto/template]: Didn't see its an update. Revision needs to be set to 1

**Comment by Hoshpak** [comment]
Did you push force-push based on an outdated local branch by chance? Now the user account in the service script seems to be wrong again.

**Comment by damadmai** [comment]
Yes it did exactly that inadvertantly. Thanks for looking over it so precisely!

**Pull request merged**